Dockerfile.ubuntu

```diff
@@ -117,6 +117,42 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
     python3 -m pip install "numpy<2.0.0" --no-cache-dir  && \
     python3 -m pip install "Jinja2==3.1.4" --no-cache-dir
 
+# begin section for GPU testing in build img
+RUN apt-get update ; \
+    apt-get install -y --no-install-recommends curl ca-certificates libnuma1 ocl-icd-libopencl1 && \
+    rm -rf /var/lib/apt/lists/* && \
+        case $INSTALL_DRIVER_VERSION in \
+        "23.22.26516") \
+                mkdir /tmp/gpu_deps && cd /tmp/gpu_deps ; \
+                curl -L -O https://github.com/intel/compute-runtime/releases/download/23.22.26516.18/intel-level-zero-gpu_1.3.26516.18_amd64.deb ; \
+                curl -L -O https://github.com/intel/intel-graphics-compiler/releases/download/igc-1.0.14062.11/intel-igc-core_1.0.14062.11_amd64.deb ; \
+                curl -L -O https://github.com/intel/intel-graphics-compiler/releases/download/igc-1.0.14062.11/intel-igc-opencl_1.0.14062.11_amd64.deb ; \
+                curl -L -O https://github.com/intel/compute-runtime/releases/download/23.22.26516.18/intel-opencl-icd_23.22.26516.18_amd64.deb ; \
+                curl -L -O https://github.com/intel/compute-runtime/releases/download/23.22.26516.18/libigdgmm12_22.3.0_amd64.deb ; \
+                dpkg -i *.deb && rm -Rf /tmp/gpu_deps ; \
+        ;; \
+        *) \
```

Comment:
not needed version configuration. In build image a single version is sufficient. In release image the old versions are just in case some unexpected issues on unsupported old platforms